providers/common/io/src/airflow/providers/common/io/xcom/backend.py

```diff
@@ -149,7 +145,7 @@ def serialize_value(  # type: ignore[override]
         return BaseXCom.serialize_value(str(p))
 
     @staticmethod
-    def deserialize_value(result: XCom) -> Any:
+    def deserialize_value(result) -> Any:
```

Comment:
Is there no typing we can usefully have anymore here?

Reply:
Slightly difficult to do that right now, I will revisit it